src/trans/internal/trltog_mod.F90

```diff
@@ -154,7 +154,7 @@ SUBROUTINE TRLTOG_PROLOG(KF_FS,KF_GP,KVSET,&
 !     ------------------------------------------------------------------
 
 USE PARKIND1  ,ONLY : JPIM     ,JPRB
-USE YOMHOOK   ,ONLY : LHOOK,   DR_HOOK, JPHOOK
```

Comment:
Same. DR_HOOK and JPHOOK are used neither.

Reply:
Done